### ISSUE TITLE
Gatsby build error

### ISSUE THREAD

**Issue by edsu** [umd-mith/mith-static#12]
`npm run build` and `npm run start` are throwing an error due to something about how nodes are getting created? 

```
"gatsby-transformer-json" threw an error while running the onCreateNode
lifecycle:

Plugins creating nodes can not set data on the reserved field "fields"
      as this is reserved for plugins which wish to extend your nodes.

```

**Comment by edsu** [comment]
It is failing here in GitHub Action land too:

https://github.com/umd-mith/mith-static/runs/3115069012?check_suite_focus=true

**Comment by raffazizzi** [comment]
After my fix for #10, `persist.js` was outputting too much of the AirTable raw data for research items. https://github.com/umd-mith/mith-static/commit/c89a8c43fea367343b5fb13b2ec27f6fee73bd6f should fix it.

**Comment by edsu** [comment]
Thanks for the quick attention @raffazizzi ; I hope everything is going well at home! :baby_bottle:
